internal/telemetry/client.go

```diff
@@ -389,48 +427,115 @@ func (c *Client) newRequest(t RequestType) *Request {
 			LanguageVersion: runtime.Version(),
 		},
 		Host: Host{
-			Hostname:    hostname,
-			ContainerID: internal.ContainerID(),
```

Comment:
What happened to `ContainerID`?

Reply:
`ContainerID` was removed in the v2 update and should be sent sent as a header [here](https://github.com/DataDog/dd-trace-go/blob/19f0596599a3003c655c887624eac2c57d7956f3/internal/telemetry/client.go#LL451-L451C58)